drivers/overlay/composefs_supported.go

```diff
@@ -119,7 +125,13 @@ func generateComposeFsBlob(toc []byte, destFile string) error {
 	return nil
 }
 
-func mountErofsBlob(blobFile, mountPoint string) error {
+func mountComposefsBlob(dataDir, mountPoint string) error {
+	data, err := getComposefsAdditionalData(dataDir)
```

Comment:
What is this needed for?

Reply:
Ah, its removed in a later commit. Maybe should drop it?

Reply:
yes, cleaned up the patches